lib/module/src/module.rs

```diff
@@ -463,7 +463,7 @@ where
     pub fn define_function(&mut self, func: FuncId, ctx: &mut Context) -> ModuleResult<()> {
         let compiled = {
             let code_size = ctx.compile(self.backend.isa()).map_err(|e| {
-                dbg!(
+                warn!(
```

Comment:
Bikeshed: warn "Designates hazardous situations.", so this message seems more like an info.